### PR TITLE
docs(shell-panel): Update css var descriptions

### PR DESCRIPTION
**Pull request by macandcheese**
**Related Issue:** No related issue

## Summary
Clarifies usage of newly added CSS variables and updates verbiage to match other components. cc @geospatialem

### CODE REVIEW

File: src/components/shell-panel/shell-panel.scss [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
- * @prop --calcite-shell-panel-height: The height of the component.
- * @prop --calcite-shell-panel-max-height: The maximum height of the component.
- * @prop --calcite-shell-panel-min-height: The minimum height of the component.
- * @prop --calcite-shell-panel-detached-max-height: The maximum height of the component when `detached` is true.
+ * @prop --calcite-shell-panel-width: Specifies the width of the component.
+ * @prop --calcite-shell-panel-max-width: Specifies the maximum width of the component.
+ * @prop --calcite-shell-panel-min-width: Specifies the minimum width of the component.
+ * @prop --calcite-shell-panel-height: When `layout` is `horizontal`, specifies the height of the component.
+ * @prop --calcite-shell-panel-max-height: When `layout` is `horizontal`, specifies the maximum height of the component.
+ * @prop --calcite-shell-panel-min-height: When `layout` is `horizontal`, specifies the minimum height of the component.
```
Comment (geospatialem) [inline src/components/shell-panel/shell-panel.scss:11]: nitpick:
> `horizontal` -> `"horizontal"`

File: src/components/shell-panel/shell-panel.scss [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
+ * @prop --calcite-shell-panel-detached-max-height: When `detached` is true, specifies the maximum height of the component.
```
Comment (geospatialem) [inline src/components/shell-panel/shell-panel.scss:12]: nitpick:
> true -> `true`

**Review by geospatialem — COMMENTED** [review]
Thanks for tackling so fast! A few nitpicks for consistency.

**Review by geospatialem — APPROVED** [review]
👍🏻

**Review by driskull — APPROVED** [review]
👍

**Pull request merged**